tests/kernel/gen_isr_table/src/main.c

```diff
@@ -17,6 +17,11 @@ extern uint32_t _irq_vector_table[];
 #define HAS_DIRECT_IRQS
 #endif
 
+/* ARC don't support dynamic interrupts on the systems with ARConnect */
```

Comment:
I'd add more justification here. As you put it in the PR message for example:
> We don't support dynamic interrupts for the common interrupts for SMP systems for now.
Moreover maybe add even more details, why's that.

Reply:
Done.